### ISSUE TITLE
Make accessibility field writable in raster layer form

### ISSUE THREAD

**Issue by lexvand** [nens/lizard-management-client#776]
Users should be able to change the accessibility of a new or existing raster layer. Currently this is prevented in the management client. When registering a new layer based on a single new source it is still good to use the access_modifier of the source as the default for the layer.

This change does not result in authorization conflicts, as the accessibility of the layer overrules that of underlying sources. Administrators won't be able to use sources they do not have access to, because they won't be able to look up the UUID of those sources.

NB: Does not apply to scenario raster layers, for which the accessibility is (going to be) managed through the scenario.

**Comment by lexvand** [comment]
Hi @hoanphungt,

Could you include this change in the branch for the upcoming production release? This is currently experienced as a major blocking issue by some users. Has been discussed with Joeri.

**Comment by lexvand** [comment]
@hoanphungt I see you didn't require a review for these changes. If you feel confident that it works correctly it's okay by me to release directly to production.

**Comment by hoanphungt** [comment]
@lexvand 
Ah yes, I deployed it to staging and tested it again. Worked correctly and I deployed it to production already. Gave staging back to the timeseries management. Could you have a look again?

I also created this issue while testing it: https://github.com/nens/lizard-management-client/issues/783

**Comment by lexvand** [comment]
Great work and good thinking!
